hashedindex/textparser.py

```diff
@@ -52,17 +50,6 @@ def tfidf(tf, df, corpus_size):
         return 0.0
 
 
-def normalize_unicode(text):
```

Comment:
This was removed partly to allow the test coverage report to succeed, which isn't a great justification!  If there's a preferred approach here, just let me know.

Reply:
Seems like this is unused anyway? Probably its usage was removed in some previous cleanup but the function did not get cleaned up along with it.

I'm pretty sure I didnt originally intend for it to be part of the public facing API.

Happy to remove it since its no longer used by anything